README.md

```diff
@@ -65,7 +65,7 @@ DIR=$(mktemp -d /tmp/dbXXX)
 ./cockroach create-ca-cert
 ./cockroach create-node-cert 127.0.0.1 localhost $(hostname)
 # Initialize data directories.
-./cockroach init $DIR
```

Comment:
there's another part further down in this file that should need the same change.